src/data/bmg_plate.py

```diff
@@ -176,3 +183,65 @@ def parse_bmg_files_from_dir(dir: str) -> tuple[pd.DataFrame, np.ndarray]:
     df = pd.DataFrame(plate_summaries)
     plate_values = np.asarray(plate_values)
     return df, plate_values
+
+
+def calculate_activation_inhibition_zscore(
+    df_stats: pd.Series,
+    values: np.ndarray,
+    mode: str = "all",
```

Comment:
suggestion: maybe create and enum for modes so that there is less room for errors?

Reply:
Good idea, will add that 👍